eclass/kernel-build.eclass

```diff
@@ -320,6 +321,7 @@ kernel-build_src_install() {
 	# strip empty directories
 	find "${D}" -type d -empty -exec rmdir {} + || die
 
+	[[ ! -f "${ED}${kernel_dir}/include/config/kernel.release" ]] && cp "${WORKDIR}/build/include/config/kernel.release" "${ED}${kernel_dir}/include/config/"
 	local relfile=${ED}${kernel_dir}/include/config/kernel.release
```

Comment:
I think it's fine to just change `relfile` if the other path is fine for both scenarios.

Reply:
We could for this specific case, but if `kernel.release` is not installed it will cause problems in other places.

I have finally figured out what is going on with `kernel.release` and was able to avoid this ugly check-and-copy. In short, with modules, `kernel.release` ends up in `modprep` directory and is then installed from there. That does not happen without modules.